CribblyBackend.UnitTests/ControllerTests/TeamControllerTests.cs

```diff
@@ -122,5 +122,21 @@ public async Task Create_ShouldReturn500_IfError()
             var typedResult = Assert.IsType<ObjectResult>(result);
             Assert.Equal(StatusCodes.Status500InternalServerError, typedResult.StatusCode);
         }
+
+        [Fact]
+        public async Task Delete_ShouldReturnOk_IfNoError()
+        {
+            mockTeamService.Setup(x => x.Delete(It.IsAny<Team>()));
+            var result = await TeamController.Delete(new Team());
+            Assert.IsType<OkResult>(result);
```

Comment:
I _think_ there might be a status code that's like "no content" or something that may be more appropriate for delete

Reply:
Changed

Reply:
Changed